sabnzbd/assembler.py

```diff
@@ -169,6 +169,11 @@ def _assemble(nzf, path, dupe):
     decodetable = nzf.decodetable
 
     for articlenum in decodetable:
+        # Break if deleted during writing
+        if nzf.nzo.deleted:
```

Comment:
OK, that's true, should be good enough.
The is_gone() function would test for a series of Statuses, I assume?